src/Http/Http.Extensions/src/HttpContextJsonExtensions.Response.cs

```diff
@@ -0,0 +1,80 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Text.Json;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.AspNetCore.Http.Json
+{
+    public static partial class HttpContextJsonExtensions
+    {
+        public static ValueTask WriteAsJsonAsync<TValue>(
+            this HttpResponse response,
+            TValue value,
+            CancellationToken cancellationToken = default)
+        {
+            return response.WriteAsJsonAsync<TValue>(value, options: null, contentType: JsonConstants.JsonContentType, cancellationToken);
```

Comment:
With writing, the defaults need to be `Web` + need to use the relaxed encoding. See https://github.com/dotnet/aspnetcore/blob/master/src/Mvc/Mvc.Core/src/Formatters/SystemTextJsonOutputFormatter.cs#L42